flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -41,10 +41,13 @@
 #include "mlir/Transforms/RegionUtils.h"
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/Frontend/OpenMP/OMPConstants.h"
+#include "llvm/Support/CommandLine.h"
 
 using namespace Fortran::lower::omp;
 using namespace Fortran::common::openmp;
 
+static llvm::cl::opt<bool> DumpAtomicAnalysis("fdebug-dump-atomic-analysis");
```

Comment:
If we plan to commit this, wouldn't it be consistent to add this as an option to the clang driver with FC1 visibility? If this is rather just to ease debugging while the development of this patch, then it is fine.